spec/models/user_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
```

Comment:
All the tests are failing. Please make sure all the tests are green.
![tests](https://user-images.githubusercontent.com/35504149/197310387-5aff6fa5-2e9c-48ed-8a5b-291307e47696.png)

Reply:
I will look into this